purchases/src/main/kotlin/com/revenuecat/purchases/AppConfig.kt

```diff
@@ -0,0 +1,52 @@
+package com.revenuecat.purchases
+
+import android.content.Context
+import java.net.URL
+
+internal class AppConfig(
+    context: Context,
+    observerMode: Boolean,
+    val platformInfo: PlatformInfo,
+    proxyURL: URL?
+) {
+
+    val languageTag: String = context.getLocale()?.toBCP47() ?: ""
+    val versionName: String = context.versionName ?: ""
+    var finishTransactions: Boolean = !observerMode
+    val baseURL: URL = proxyURL?.also {
+        debugLog("Purchases is being configured using a proxy for RevenueCat")
```

Comment:
will this be output every time the variable is read?

Reply:
Only when set, `baseURL` will be set when the constructor is called